man/fitNullModel.Rd

```diff
@@ -112,11 +111,25 @@ isNullModelSmall(null.model)
     \item{converged}{A logical indicator of whether the AIREML procedure for estimating the random effects variance components converged.}
     \item{zeroFLAG}{A vector of logicals the same length as \code{varComp} specifying whether the corresponding variance component estimate was set to 0 by the function due to convergence to the boundary in the AIREML procedure.}
     \item{RSS}{The residual sum of squares from the model fit. When \code{family} is gaussian, this will typically be 1 since the residual variance component is estimated separately.}
-    \item{sample.id}{A vector of IDs for the samples used in the
-      analysis.}
+    \item{model}{The model string.}
+    \item{outcome}{The outcome variable name.}
```

Comment:
@amstilp is there any reason to be concerned that `outcome` used to be the actual outcome vector, and now it's just a string for the outcome variable?

Reply:
I thought about this briefly and decided not to worry about it, but I could be convinced. The only alternative I could think of to `outcome` would be `outcome_string`, but then we should change the other elements to `model_string` and `covar_strings` (which isn't really a string; it's a list of strings). So I stuck with `outcome` and figured that users would read the help or the `NEWS` if they noticed a difference.

Reply:
I was trying to look at what other functions do (e.g. `lm`, `lmer`) if anything. Had another idea that I thought I'd run by you. What if we had an object in the output called `model` that is a list with the elements `formula` (currently model), `outcome`, and `covars`. Perhaps we would also move `family` and `hetResid` into that list. Basically all the things that describe the model and aren't data. What do you think? Better or worse?

Reply:
I think that makes sense.

Reply:
I made this change, including moving `hetResid` and `family` to the new `model` list element.